bugbot/rules/has_str_no_range.py

```diff
@@ -109,7 +109,7 @@ def get_bz_params(self, date):
             "n6": 1,
             "f6": "reporter",
             "o6": "regexp",
-            "v6": r"^.*@softvision\.(ro|com)$",
+            "v6": r"^.*@mozilla\.com$",
```

Comment:
Would this not exclude too many people? At the same time, it will not exclude people who do not use their Mozilla emails for Bugzilla.

Reply:
I think the has_str_no_range tool was mostly useful for external users, and the problem only appears for QA (which don't use their personal emails), so all in all this should be OK